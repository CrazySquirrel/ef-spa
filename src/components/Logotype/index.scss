.logotype {
  box-sizing: border-box;
  cursor: pointer;
  display: block;
  fill: #02b3e4;
  height: 64px;
  padding: 8px;
  transition: fill .2s;
  width: 64px;

  svg {
    cursor: pointer;
  }
}
