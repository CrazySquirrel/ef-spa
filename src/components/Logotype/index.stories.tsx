import * as React from 'react';

import {storiesOf} from '@storybook/react';

import Logotype from './index';

storiesOf('Logotype', module)
    .add('default', () => (
        <Logotype/>
    ));
