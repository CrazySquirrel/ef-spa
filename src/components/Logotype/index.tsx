import * as React from 'react';
import * as bem from 'bem-cn';

import Link from 'components/Link';

import {IconChat} from 'components/Icon';

import './index.scss';

const block = bem('logotype');

export default class Logotype extends React.Component<{}, {}> {
  public render() {
    return (
        <Link to='/' replaceClassName={block()}>{IconChat()}</Link>
    );
  }
}
