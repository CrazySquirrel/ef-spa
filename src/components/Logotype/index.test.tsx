import * as React from 'react';
import {shallow} from 'enzyme';

import Logotype from './index';

describe('Logotype', () => {
  it('render', () => {
    const result = shallow(<Logotype/>);

    expect(result).toMatchSnapshot();
  });
});
