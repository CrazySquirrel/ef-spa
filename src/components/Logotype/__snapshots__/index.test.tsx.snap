version https://git-lfs.github.com/spec/v1
oid sha256:0e6b729d2ada210c6710734ccedb493b438a4490652b7d1a9dd1a9a51d8d3f1f
size 2892
